app/models/manageiq/providers/openshift/container_manager/refresh_worker/watch_thread.rb

```diff
@@ -1,2 +1,12 @@
 class ManageIQ::Providers::Openshift::ContainerManager::RefreshWorker::WatchThread < ManageIQ::Providers::Kubernetes::ContainerManager::RefreshWorker::WatchThread
+  def self.start!(ems, queue, entity_type, resource_version)
+    connect_options = ems.connect_options
+    ems_klass       = ems.class
+
+    # Handle openshift entity types with different API paths
+    connect_options[:api_group] = ems_klass.api_group_for_kind(entity_type.camelize.singularize)
```

Comment:
should the `entity_type.camelize.singularize` transformation be done inside the `api_group_for_kind` method so that this will read as `connect_options[:api_group] = ems_klass.api_group_for_kind(entity_type)`

Reply:
I now see that the `api_group_for_kind` method already existed as an instance method, and this PR just makes it a class method.

Reply:
Yeah that method is intended to take the `:kind` attribute of a k8s object e.g. https://github.com/ManageIQ/manageiq-providers-openshift/blob/master/app/models/manageiq/providers/openshift/container_manager/container_template.rb#L37